pkg/report/timeseries.go

```diff
@@ -128,9 +128,9 @@ func (ts TimeSeries) String() string {
 	for i := range ts {
 		row := []string{
 			fmt.Sprintf("%d", ts[i].Timestamp),
-			fmt.Sprintf("%s", ts[i].MinLatency),
-			fmt.Sprintf("%s", ts[i].AvgLatency),
-			fmt.Sprintf("%s", ts[i].MaxLatency),
+			fmt.Sprintf("%v", ts[i].MinLatency),
```

Comment:
the gosimple complains `should use String() instead of fmt.Sprintf (S1025)`. Is there a reason not to use `ts[i].MinLatency.String()`? Just wondering. same for `ts[i].AvgLatency` and `ts[i].MaxLatency`.